src/SocketIOHandler.ts

```diff
@@ -137,6 +155,38 @@ class SocketIOHandler {
     return token;
   }
 
+  /**
+   * Register a callback in Kafka Messenger events.
+   * @param subject Kafka subject
+   * @param event
+   * @param callback callback to be registered for Kafka Messenger
+   * @param token the token returned by the Socket
+   */
+  private registerCallback(subject: string, event: string, cb: (ten: string, data: any) => void, token: string): void {
```

Comment:
It seems that the handlers will be registered at the Kafka Messenger only when the first socket.io connection is established. But, it seems that the callbacks are removed when the socket.io connection that cause the registration is closed. What happens with other socket.io connections that are still opened?  Shouldn't the callback unregistered when the last socket.io connection is closed?

Reply:
From what I tested, the scenarios for SocketIO connection opening that we have in the current Dojot version are:
1) Clicking in notifications
2) Clicking in a device to see its dashboard
3) Clicking in a attribute that is a geopoint

In 1) we have only one connection, that will end when we exit from the notifications.
In 3) we open a connection when we already have the connection 2) open, and the connection to the map will close when we leave the device's dashboard, so there will be no connections that need the handlers.

Reply:
That's right for one browser, but if a open multiple browsers?

Reply:
Haven't thought about it. I will test and modify if necessary, thanks!

Reply:
I think you can implement some kind of reference count with few changes.